src/modules/features/selectors.ts

```diff
@@ -163,6 +163,7 @@ export const isCreditsFeatureEnabled = (
   }
 
   const walletsAllowed = userWalletsVariant.payload.value
+    .trim()
```

Comment:
Should we also trim the wallet just in case? I don't remember where you've done this already